coderd/database/migrations/000373_task_status_notifications.up.sql

```diff
@@ -0,0 +1,63 @@
+-- Task transition to 'working' status
+INSERT INTO notification_templates (
+	id,
+	name,
+	title_template,
+	body_template,
+	actions,
+	"group",
+	method,
+	kind,
+	enabled_by_default
+) VALUES (
+			 'bd4b7168-d05e-4e19-ad0f-3593b77aa90f',
+			 'Task Working',
```

Comment:
I kept the terminology aligned with the current workspace agent app status (`Working`, `Idle`). In PR: https://github.com/coder/coder/pull/19773/files#diff-1894d4f2ccb1125739e504fc6d8d3a7daeec9dd829a81c342cf28f3e92efbf58R2-R7 I see the task status types differ. Is that finalized? If yes, should I update this PR to adopt the new task status terms?